browser/web_discovery/web_discovery_utils.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_WEB_DISCOVERY_WEB_DISCOVERY_UTILS_H_
+#define BRAVE_BROWSER_WEB_DISCOVERY_WEB_DISCOVERY_UTILS_H_
+
+class PrefService;
+class TemplateURLService;
+
+bool IsBraveSearchDefault(TemplateURLService* template_service);
+bool ShouldCreateWebDiscoveryTabHelper(PrefService* prefs);
+// Returns true when increased.
+bool IncreaseBraveSearchVisitCount(PrefService* prefs,
```

Comment:
You can choose to ignore this, but code-wise it seems weird for this method to (1) do any conditional checking and (2) return if it succeeded. It doesn't seem unreasonable that the caller verifies brave search is default and web discovery is enabled before calling this method. It looks like a lot of your other utility methods already do these same checks.

If you want to keep it as-is, then change the name to `MaybeIncreaseBraveSearchVisitCount`.